### PR TITLE
Add Samo INU Token

### PR DESCRIPTION
**Pull request by SamoInu**
## I agree to not ping _anybody_ on Discord/Twitter/email about this pull request. Instead I will inquire by posting a new comment in the pull request if needed.

---

PRs are reviewed in bulk and and can take up to **two weeks** to be merged.

_This repository is managed using an auto merge action. Please ensure your PR has no deleted lines, and it will be merged._

## **Please provide the following information for your token.**

Please include change to the `src/tokens/solana.tokenlist.json` file in the PR.
DON'T modify any other token on the list.

At minimum each entry should have

- Token Address:
- Token Name:
- Token Symbol:
- Logo: (logo should be uploaded under assets/mainnet/<mint address>/\*.<png/svg>)
- Link to the official homepage of token:
- Coingecko ID if available (https://www.coingecko.com/api/documentations/v3#/coins/get_coins__id_):

### CODE REVIEW

File: src/tokens/solana.tokenlist.json [hunk 1]
```diff
@@ -21046,7 +21046,22 @@
         "website": ""        
       }
     }
+  ,{
+  "chainId": 101,
+  "address": "F7mgxaYF1gg1hBtaVzENSG6ey3pn6J1mXhBRmHxDzBNg ",
+  "symbol": "SINU ",
+  "name": "Samo INU ",
```
Comment (ebramanti) [inline src/tokens/solana.tokenlist.json:21053]: This space is causing issues in address parsing, needs to be addressed. Spaces also in symbol/name as well.

Comment (SamoInu) [reply src/tokens/solana.tokenlist.json:21053]: Hello ebramanti,

please how can this be corrected?

Thans

**Comment by fanatid** [comment]
What is the difference between this PR and https://github.com/solana-labs/token-list/pull/2710 https://github.com/solana-labs/token-list/pull/2623 https://github.com/solana-labs/token-list/pull/2621 https://github.com/solana-labs/token-list/pull/2617 https://github.com/solana-labs/token-list/pull/2585 https://github.com/solana-labs/token-list/pull/2464 ?

File: src/tokens/solana.tokenlist.json (hunk 1 above)
Comment (ebramanti) [reply src/tokens/solana.tokenlist.json:21053]: Remove trailing spaces at the end of all of these lines

**Pull request merged**